src/apps/toolbar/modules/Notifications/infra/ArrivalPreview.tsx

```diff
@@ -0,0 +1,93 @@
+import { invoke } from '@tauri-apps/api/core';
+import { Button } from 'antd';
+import { AnimatePresence, motion } from 'framer-motion';
+import moment from 'moment';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { SeelenCommand } from 'seelen-core';
+
+import { BackgroundByLayersV2 } from '../../../../seelenweg/components/BackgroundByLayers/infra';
+
+import { Selectors } from '../../shared/store/app';
+
+import { Icon } from '../../../../shared/components/Icon';
+
+// Difference between Windows epoch (1601) and Unix epoch (1970) in milliseconds
+const EPOCH_DIFF_MILLISECONDS = 11644473600000n;
+
+function WindowsDateFileTimeToDate(fileTime: bigint) {
+  return new Date(Number(fileTime / 10000n - EPOCH_DIFF_MILLISECONDS));
+}
+
+export function ArrivalPreview() {
+  const notifications = useSelector(Selectors.notifications);
+  const [ currentNotificationPreviewSet, setCurrentNotificationPreviewSet ] = useState(false);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setCurrentNotificationPreviewSet(notifications.filter((notification) => {
+        const arrivalDate = WindowsDateFileTimeToDate(BigInt(notification.date));
+
+        return moment(Date.now()).diff(arrivalDate, 'seconds') < 10;
+      }));
+    }, 10 * 1000);
+
+    // Cleanup function to clear the timeout if the component unmounts
+    return () => clearTimeout(timeoutId);
+  }, [currentNotificationPreviewSet]);
+
+  useEffect(() => {
+    setCurrentNotificationPreviewSet(notifications.filter((notification) => {
+      const arrivalDate = WindowsDateFileTimeToDate(BigInt(notification.date));
+
+      return moment(Date.now()).diff(arrivalDate, 'seconds') < 10;
+    }));
+  }, [notifications]);
+
+  return (
+    <BackgroundByLayersV2 className="notification-arrival-preview">
+      <div className="notifications-body">
```

Comment:
rename classname to just `notification-arrival` and remove the internal div "notifications-body" it's not needed you can use `notification-arrival` class I think.